scene/2d/joint_2d.cpp

```diff
@@ -289,11 +297,103 @@ real_t PinJoint2D::get_softness() const {
 	return softness;
 }
 
+void PinJoint2D::set_angular_limit_lower(real_t p_angular_limit_lower) {
+	if (angular_limit_lower == p_angular_limit_lower) {
+		return;
+	}
+	angular_limit_lower = p_angular_limit_lower;
+	queue_redraw();
+	if (is_configured()) {
+		PhysicsServer2D::get_singleton()->pin_joint_set_param(get_joint(), PhysicsServer2D::PIN_JOINT_LIMIT_LOWER, p_angular_limit_lower);
```

Comment:
```suggestion
		PhysicsServer2D::get_singleton()->pin_joint_set_param(get_rid(), PhysicsServer2D::PIN_JOINT_LIMIT_LOWER, p_angular_limit_lower);
```
After #80736

Reply:
Uf, was wondering why build was failing now :D Thanks. This saves time.